Cargo.toml

```diff
@@ -44,6 +44,8 @@ resolver = "2"
 # temporary solution to funty@1.2.0 being yanked, we should propose bitvec upgrade to upstream filecoin crates
 # tracking issue: https://github.com/bitvecto-rs/funty/issues/7
 funty = { git = "https://github.com/bitvecto-rs/funty/", rev = "7ef0d890fbcd8b3def1635ac1a877fc298488446" }
+# get rid of RUSTSEC-2020-0071, the fix is in place but not published to crates.io
```

Comment:
Could add a link to the tracking issue here.

Reply:
Done